README.md

```diff
@@ -16,6 +16,10 @@ Write a log message in the #yoga channel.
 - Install dependencies `yarn`
 - Start the bot `yarn dev`
 
+## 🤖 How to deploy
+
+- Need to run `node ./src/deploy-commands.js` to deploy the slash commands to all connected Discord servers.
```

Comment:
Github Copilot suggested the title, and even though there are no other deploy-instructions I thought it looked nice 🙂